V/vmecpp_julia/build_tarballs.jl

```diff
@@ -0,0 +1,267 @@
+sed -i 's/"-march=armv8-a+crypto"//g' abseil-cpp/abseil-cpp/absl/copts/GENERATED_AbseilCopts.cmake
+sed -i 's/"-maes"//g' abseil-cpp/abseil-cpp/absl/copts/GENERATED_AbseilCopts.cmake
+sed -i 's/"-msse4.1"//g' abseil-cpp/abseil-cpp/absl/copts/GENERATED_AbseilCopts.cmake
+sed -i 's/"-mfpu=neon"//g' abseil-cpp/abseil-cpp/absl/copts/GENERATED_AbseilCopts.cmake
+
+mkdir -p abseil-build && cd abseil-build
+cmake ../abseil-cpp/abseil-cpp \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=OFF \
+    -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+    -DCMAKE_CXX_STANDARD=20 \
+    -DABSL_PROPAGATE_CXX_STD=ON \
+    -DABSL_BUILD_TESTING=OFF
+make -j${nproc}
+make install
+cd ..
+
+# ============================================
+# Step 2: Build vmecpp core (static library)
+# ============================================
+echo "Building vmecpp core..."
+
+# Patch vmecpp CMakeLists.txt to remove Python bindings (pybind11)
+# We only need vmecpp_core, not the Python module or indata2json executable
+sed -i '/FetchContent_Declare.*pybind11/,/FetchContent_MakeAvailable.*pybind11/d' vmecpp/CMakeLists.txt
+sed -i '/pybind11_add_module/,/install.*_vmecpp/d' vmecpp/CMakeLists.txt
+# Also remove install target for indata2json (we only build vmecpp_core)
+sed -i '/install.*TARGETS.*indata2json/d' vmecpp/CMakeLists.txt
+
+mkdir -p vmecpp-build && cd vmecpp-build
+
+# macOS: SDK 12.3 has full C++20 support, but we need to disable availability annotations
+# since we're targeting deployment_target=10.15 but using features from SDK 12.3
+MACOS_CXX_FLAGS=""
+if [[ "${target}" == *-apple-* ]]; then
+    echo "macOS detected: disabling availability annotations (using SDK 12.3 with deployment target 10.15)"
+    MACOS_CXX_FLAGS="-D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+# Configure vmecpp with vendored dependencies
+# Set BLAS/LAPACK to use OpenBLAS from JLL
+# Note: FetchContent variable names use the EXACT name from FetchContent_Declare
+# For packages with hyphens, CMake converts them to underscores in cache variables
+# BUT we also need to try the hyphenated form for compatibility
+cmake ../vmecpp \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_STANDARD=20 \
+    -DCMAKE_CXX_FLAGS="${MACOS_CXX_FLAGS}" \
+    -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+    -DFETCHCONTENT_SOURCE_DIR_EIGEN=${WORKSPACE}/srcdir/eigen/eigen \
+    -DFETCHCONTENT_SOURCE_DIR_NLOHMANN_JSON=${WORKSPACE}/srcdir/nlohmann_json/json \
+    -DFETCHCONTENT_SOURCE_DIR_ABSEIL-CPP=${WORKSPACE}/srcdir/abseil-cpp/abseil-cpp \
+    -DFETCHCONTENT_SOURCE_DIR_ABSCAB-CPP=${WORKSPACE}/srcdir/abscab-cpp/abscab-cpp \
+    -DFETCHCONTENT_SOURCE_DIR_INDATA2JSON=${WORKSPACE}/srcdir/indata2json/indata2json \
+    -DFETCHCONTENT_FULLY_DISCONNECTED=ON \
+    -Dabsl_DIR=${prefix}/lib/cmake/absl \
+    -DBLA_VENDOR=OpenBLAS \
```

Comment:
If possible, I would suggest trying to build with libblastrampoline instead of OpenBLAS here, because that would allow users to choose a different BLAS library for their system.